hibernate-core/src/main/java/org/hibernate/hql/internal/ast/tree/DotNode.java

```diff
@@ -690,6 +697,25 @@ public Type getDataType() {
 		return super.getDataType();
 	}
 
+	@Override
+	public String[] getReferencedTables() {
+		String[] referencedTables = null;
+		AST firstChild = getFirstChild();
+		if ( firstChild != null ) {
+			if ( firstChild instanceof FromReferenceNode ) {
+				FromReferenceNode fromReferenceNode = (FromReferenceNode) firstChild;
+				FromElement fromElement = fromReferenceNode.getFromElement();
+				if ( fromElement != null ) {
+					String table = fromElement.getPropertyTableName( getOriginalPropertyName() );
+					if (table != null) {
```

Comment:
Missing padding. Looks like our Checkstyle rules are not correctly enforced.

Reply:
Fixed. I must set automatic formatting in IntelliJIdea properly...